cider/exec/nextgen/jitlib/base/JITTuple.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef JITLIB_BASE_JITTUPLE_H
+#define JITLIB_BASE_JITTUPLE_H
+
+#include <boost/container/small_vector.hpp>
+
+#include <cstddef>
+
+#include "exec/nextgen/jitlib/base/JITValue.h"
+
+namespace cider::jitlib {
+class JITTuple final : public JITBaseValue {
+ public:
+  JITTuple(JITTypeTag type_tag, JITBackendTag backend)
+      : JITBaseValue(type_tag, backend){};
+
+  template <typename ChildValueTpye = JITValue>
+  ChildValueTpye& get_as(size_t index) const {
```

Comment:
Fix function naming. I think ```getElementAs``` is a good option.

Reply:
get it